.travis.yml

```diff
@@ -2,6 +2,9 @@ language: ruby
 rvm:
   - 2.3.0
 script:
+  - bin/rake db:create RAILS_ENV=test
   - bin/rake db:migrate RAILS_ENV=test
   - bundle exec rspec --color --format documentation
   - bundle exec rubocop
+before_script:
+  - cp config/database.yml.travis config/database.yml
```

Comment:
Oh I see, that makes sense. Maybe we should just read in the DB connection stuff from the environment (like on Heroku)?